core/drivers/clk/clk-stm32mp25.c

```diff
@@ -55,6 +64,34 @@
 #define RCC_4_MHZ	UL(4000000)
 #define RCC_16_MHZ	UL(16000000)
 
+/*
+ * CIDCFGR register bitfields
+ */
+#define RCC_CIDCFGR_SEMWL_MASK		GENMASK_32(23, 16)
+#define RCC_CIDCFGR_SCID_MASK		GENMASK_32(6, 4)
+#define RCC_CIDCFGR_CONF_MASK		(_CIDCFGR_CFEN |	\
+					 _CIDCFGR_SEMEN |	\
+					 RCC_CIDCFGR_SCID_MASK |\
+					 RCC_CIDCFGR_SEMWL_MASK)
+
+/*
+ * SECCFGR register bitfields
+ */
+#define RCC_SECCFGR_EN			BIT(0)
+
+/*
+ * SEMCR register bitfields
+ */
+#define RCC_SEMCR_SCID_MASK		GENMASK_32(6, 4)
+#define RCC_SEMCR_SCID_SHIFT		U(4)
+
+/*
+ * RIF miscellaneous
+ */
+
```

Comment:
Remove this empty line.
By the way, prefer One line inline comments at lines 68, 78, 83 and 89.